package.json

```diff
@@ -9,9 +9,12 @@
     "start:android:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'react-native run-android'",
     "start:ios": "concurrently -r 'react-native start' 'react-native run-ios'",
     "start:ios:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'react-native run-ios'",
+    "build:android:release": "cd android && ./gradlew assembleRelease",
     "link": "react-native link",
     "ios": "react-native run-ios",
     "android": "react-native run-android",
+    "android:release": "react-native run-android --variant=release",
+    "android:signkey": "cd android/app/ && keytool -genkey -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000 && cd -",
```

Comment:
Nope I probably wouldn't need to run the command since I have a key but it simplifies the process. Was curious about what @Antoine38660 meant by adding an env variable but it should be good.